tests/cross/src/test/scala/tests/pc/CompletionOverrideAllSuite.scala

```diff
@@ -0,0 +1,362 @@
+package tests.pc
+
+import tests.BaseCompletionSuite
+
+object CompletionOverrideAllSuite extends BaseCompletionSuite {
+
+  override def beforeAll(): Unit = {
+    indexJDK()
+  }
+
+  check(
+    "simple",
+    """|package example
+       |
+       |trait Foo {
+       |  def foo: Int
+       |  def bar: Int
+       |}
+       |object Main {
+       |  val x = new Foo {
+       |    def@@
+       |  }
+       |}
+       |""".stripMargin,
+    """|Implement all members (2 total)
+       |""".stripMargin,
+    filter = _.contains("Implement")
+  )
+
+  checkEdit(
+    "simple-edit",
+    """|package example
+       |
+       |trait Foo {
+       |  def foo: Int
+       |  def bar: Int
+       |}
+       |object Main {
+       |  val x = new Foo {
+       |    def@@
+       |  }
+       |}
+       |""".stripMargin,
+    """|package example
+       |
+       |trait Foo {
+       |  def foo: Int
+       |  def bar: Int
+       |}
+       |object Main {
+       |  val x = new Foo {
+       |    def foo: Int = ${0:???}
+       |def bar: Int = ${0:???}
```

Comment:
So the formatting is a bit odd in these. I'm assuming because either range or type formatting isn't being triggered when these lines are being put in here? I'm not sure if I should just rely on when you're in the editor and the line is put there that it will then automatically be formated by (I think the range formatting?), or if I need to also calculate the amount of indent there should be there?

Reply:
I would expect the completions to include the indentation here and I think we already calculate the necessary indentation for similar features.